Neos.FluidAdaptor/Classes/Core/ViewHelper/ViewHelperResolver.php

```diff
@@ -53,6 +53,12 @@ class ViewHelperResolver extends \TYPO3Fluid\Fluid\Core\ViewHelper\ViewHelperRes
      */
     protected $namespaces = [];
 
+    /**
+     * @Flow\InjectConfiguration(path="defaultNamespaces")
+     * @var array
+     */
+    protected $defaultNamespacesFromConfiguration;
```

Comment:
@martinlipp The `defaultNamespacesFromConfiguration` could/should be renamed to `configuredNamespaces` or similar - they are not default but are added from configuration.

Reply:
I'd prefer `namespacesFromConfiguration`. It's less ambiguous than `configuredNamespaces`.

Reply:
Agree - Please update the PR :)